Add service startup message

anovakovic01 opened the pull request:
Add startup message to manager, http, coap and normalizer services.

Signed-off-by: Aleksandar Novakovic <anovakovic01@gmail.com>

anovakovic01 committed:
Add service startup message

Add startup message to manager, http, coap and normalizer services.

Signed-off-by: Aleksandar Novakovic <anovakovic01@gmail.com>

mijicd commented on cmd/http/main.go:
```diff
@@ -75,6 +75,8 @@ func main() {
 		p := fmt.Sprintf(":%s", cfg.Port)
 		mc := manager.NewClient(cfg.ManagerURL)
+		startMsg := fmt.Sprintf("HTTP adapter service started, exposed port %s", cfg.Port)
+		logger.Log("info", startMsg)
```
Please replace `info` with `message`.

mijicd commented on cmd/manager/main.go:
```diff
@@ -94,6 +94,8 @@ func main() {
 	go func() {
 		p := fmt.Sprintf(":%s", cfg.Port)
+		startMsg := fmt.Sprintf("Manager service started, exposed port %s", cfg.Port)
+		logger.Log("info", startMsg)
```
Please replace `info` with `message`.

mijicd commented on cmd/normalizer/main.go:
```diff
@@ -45,6 +45,8 @@ func main() {
 	go func() {
 		p := fmt.Sprintf(":%s", cfg.Port)
+		startMsg := fmt.Sprintf("Normalizer service started, exposed port %s", cfg.Port)
+		logger.Log("info", startMsg)
```
Please replace `info` with `message`.

mijicd commented on cmd/http/main.go:
```diff
@@ -75,6 +75,8 @@ func main() {
 	go func() {
 		p := fmt.Sprintf(":%s", cfg.Port)
 		mc := manager.NewClient(cfg.ManagerURL)
+		startMsg := fmt.Sprintf("HTTP adapter service started, exposed port %s", cfg.Port)
```
Please drop the local variable.

mijicd commented on cmd/manager/main.go:
```diff
@@ -94,6 +94,8 @@ func main() {
 
 	go func() {
 		p := fmt.Sprintf(":%s", cfg.Port)
+		startMsg := fmt.Sprintf("Manager service started, exposed port %s", cfg.Port)
```
Please drop the local variable.

mijicd commented on cmd/normalizer/main.go:
```diff
@@ -45,6 +45,8 @@ func main() {
 
 	go func() {
 		p := fmt.Sprintf(":%s", cfg.Port)
+		startMsg := fmt.Sprintf("Normalizer service started, exposed port %s", cfg.Port)
```
Please drop the local variable.

mijicd commented on cmd/coap/main.go:
```diff
@@ -43,6 +43,8 @@ func main() {
 
 	go func() {
 		coapAddr := fmt.Sprintf(":%d", cfg.Port)
+		startMsg := fmt.Sprintf("CoAP adapter service started, exposed port %d", cfg.Port)
```
No need for adding startup message to CoAP adapter. @dusanb94 will ensure it is present as part of #168.

anovakovic01 committed:
Remove startup message from CoAP adapter

Remove startup message from CoAP adapter. Remove overhead startup
message variable.

Signed-off-by: Aleksandar Novakovic <anovakovic01@gmail.com>
